addon/components/base/bs-accordion/item/title.js

```diff
@@ -23,6 +23,13 @@ export default Component.extend({
    */
   collapsed: null,
 
+  /**
+   * @property disabled
+   * @type boolean
+   * @public
```

Comment:
Maybe mark this as `@private`, as it is only used internally, at users should set it at the `item`, not the `title`